docs/_docs/tech/python-package.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Python
+permalink: /docs/python-package/
+---
+
+# Weaverbird [python package](https://pypi.org/project/weaverbird/)
+
+The python module provide a way to run transformations described in a pipeline.
+Its code reside in the `server/` folder.
+
+## Purpose
+
+This package provides utility functions to translate weaverbird's pipelines into Python functions, powered by [pandas](https://pandas.pydata.org/).
+It is meant as a building block to create servers capable of understanding and executing such pipelines, and returning them to clients.
+
+## Installation
```

Comment:
Ici je m'attendais plutôt à un `pip install weaverbird` mais en fait la doc présentée correspond plutôt à qqun qui voudrait participer au dev du package python weaverbird. On pourrait peut-être avoir une section `Development` pour ça ? et c'est là qu'on ajoutera à terme le howto add a new translator

Reply:
You're right, I mixed the public docs and the contribution docs. I'll correct that right away